src/api.js

```diff
@@ -0,0 +1,28 @@
+function searchAlbums(query, page) {
+  return fetch(`https://api.discogs.com/database/search?q=${query}&type=master&key=mbubAaAXseWPUpaJLkKU&secret=TrELhUezCNdFoIfmoAdHZmfJIXljOSfW&format=vinyl&page=${page}`)
+    .then(res => {
+      if (!res.ok) {
+      throw new Error('Albums not found')
+      }
+      return res.json()})
```

Comment:
We should pull this out and make it into its own function in case we have to have another fetch call. This would make it more dynamic.
You could the call the function in the then block `.then(res => handleError(res)` 
In addition, I think the message could be made more informative. If the server where it returns a 500 error or if the album is not found a 404 error we can let the user know. This can also be discussed. `throw new Error('HTTP Error: ${res.status} -- Please try again later')`

Reply:
added new handleError function to the api file!